src/Uno.UI/UI/Xaml/Media/Animation/Animators/DispatcherAnimator.skia.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Security.Authentication.ExtendedProtection;
+using Microsoft.UI.Dispatching;
+using Windows.UI.Core;
+
+namespace Microsoft.UI.Xaml.Media.Animation
+{
+	internal abstract class DispatcherAnimator<T> : CPUBoundAnimator<T> where T : struct
+	{
+		public const int DefaultFrameRate = 60;
+
+		private Stopwatch _watch = new Stopwatch();
+		private TimeSpan? _delay;
+
+		public DispatcherAnimator(T from, T to, int frameRate = 0)
+			: base(from, to)
+		{
+		}
+
+		protected override void EnableFrameReporting() => CompositionTarget.Rendering += OnTargetFrame;
```

Comment:
I suspect that adding the handler like this, cause the animations to always be 1 frame late.